README.md

```diff
@@ -0,0 +1,67 @@
+# Winged Velociraptors
+
+A game based on One Night Ultimate Werewolf, but tailored specifically for our college friend group trying to attribute
+any card actions based on each individual's personality. 
+
+## Installs
+
+NPM Installs on Client side
+
+```
+npm install
+npm install antd 
+npm install react-router-dom
+```
+
+## Gameplay
+* 5 Minute Rounds where the Winged Velociraptors try to avoid being discovered and exposed. 
+* The Dorsal Fin Gang tries to discover the Winged Velociraptors and expose them in a majority vote
+
+## Card Types / Play Order
+These are listed in wake-up order, with 11/12 not waking up and having passive abilities
+
+1. Winged Velociraptor
```

Comment:
Should we specify how many velociraptors can be in the game? It's 3, right?

Reply:
I think we settled on 3, with the possibility of adding 1 villager of a duplicate of one card to make it easier for the raptors to blend in